lib/modules/manager/swift/artifacts.ts

```diff
@@ -0,0 +1,87 @@
+import { quote } from 'shlex';
+import { TEMPORARY_ERROR } from '../../../constants/error-messages';
+import { logger } from '../../../logger';
+import { exec } from '../../../util/exec';
+import type { ExecOptions } from '../../../util/exec/types';
+import {
+  getParentDir,
+  getSiblingFileName,
+  localPathExists,
+  readLocalFile,
+  writeLocalFile,
+} from '../../../util/fs';
+import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
+
+async function swiftPackageResolve(packageFile: string): Promise<void> {
+  const packagePath = getParentDir(packageFile);
+  const cmd = `swift package resolve --package-path ${quote(packagePath)}`;
```

Comment:
should renovate run this inside `packagePath` instead?

Reply:
From what I can tell `swift package resolve --package-path path/to/package` and `cd path/to/package; swift package resolve` are equivalent so it would be a matter of what you prefer / usually do in renovate.

Reply:
we prefer to run inside the right directory. can be simply set like here
https://github.com/renovatebot/renovate/blob/d869c946d164965393f1259db5171e2a0303fb27/lib/modules/manager/poetry/artifacts.ts#L185